todo_app/app.py

```diff
@@ -1,11 +1,17 @@
-from flask import Flask
+from flask import Flask, render_template, request, redirect, url_for
 
 from todo_app.flask_config import Config
+from todo_app.data.session_items import get_items, add_item
 
 app = Flask(__name__)
 app.config.from_object(Config())
 
-
 @app.route('/')
 def index():
-    return 'Hello World!'
+    return render_template('index.html', items=get_items())
+
+@app.post('/add-item')
```

Comment:
In all this time I've never seen this, only `app.route('/add-item', methods=["POST"])` - every day's a school day!